docs/source/naming/Introduction.rst

```diff
@@ -0,0 +1,7 @@
+Naming
+======
+
+Apptools naming is a python implementation of the naming portion of the `Java
```

Comment:
We can provide cross link to the API reference. The word "naming" also gets merged into the sentence and it is hard to see it is actually a package name.
```suggestion
:mod:`apptools.naming` package is a Python implementation of the Naming portion of the `Java
```